packages-private/sfc-playground/src/App.vue

```diff
@@ -57,7 +57,7 @@ const importMap = computed(() => {
       'vue/vapor': vapor,
     },
   }
-  if (useVaporMode.value) vaporImportMap.imports!.vue = vapor
```

Comment:
this line will cause `vue` and `vue/vapor` to use the same URL, witch will lead some bugs, see [Playground](https://vapor-repl.netlify.app/#__VAPOR__eNp9U8Fu1DAQ/ZXBHLIrrZIiOFVpBVSVKAdAgOCAOUTJbJri2JbtLItW++88O00Iq7aRImVm3sy8mXk5iDfW5ruBxbkofe06G8hzGCypSrcXUgQvxaXUXW+NC3Qgx9sNtRyuBudYhxvtQ6VrpiNtnekpQ6lshl+Z3t778yIasRPCUtcGidT7li5iyVX2jpUy9N041TzL1lJvB12Hzmiqb7n+tVofpCaKWUZxrky7enF2BthR6rIYeYMljMC9VVVgWERlIvD6B/r8xCyplBQFYmWxAIoNxkTtbdfmd95o7CK1QwbyO8Xuo41csIpzSpEYq0D49/vkC27gzeRPXR7w3/l99EnxybFnt2Mp5lioHHY6hq+/fOA9vudgb5pBAf1E8DNjL0PkOMLeDroB7QUusb1JZ+l0+9Vf7wNrPw0ViUbkMeGlwJni6h4b/R/dl/mrlIdLYIudbnif34ZeLeQU/ljG8qcp0mHGZxZV7RiX+FbBhByXWip20QnJnORE2CSsewUvQP/XW+Fd570ZdFhlzytrs/WsmbLpdtQ14Ac/yJUFHJcYZVLr6X8BxONyi7kji9IrEygqDd+p5qnmjn8BQy0nXg==)
```json
{
  "imports": {
    "vue": "https://vapor-repl.netlify.app/vue-vapor.esm-browser.js",
    "vue/server-renderer": "https://vapor-repl.netlify.app/server-renderer.esm-browser.js",
    "vue/vapor": "https://vapor-repl.netlify.app/vue-vapor.esm-browser.js"
  },
  "scopes": {}
}
```

Reply:
if the row is deleted, the `getCurrentInstance()` return value will be undefined in setup.